src/strike.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license Copyright (c) 2017, CKSource - Rémy Hubscher. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module notes/ckeditor-build/strike
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import StrikeEngine from './strikeengine';
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+import strikeIcon from '../theme/icons/strike.svg';
+
+/**
+ * The strike feature. It introduces the Strike button and the <kbd>Ctrl+E</kbd> keystroke.
+ *
+ * It uses the {@link module:basic-styles/strikeengine~StrikeEngine strike engine feature}.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class Strike extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get requires() {
+		return [ StrikeEngine ];
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'Strike';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	init() {
+		const editor = this.editor;
+		const t = editor.t;
+		const command = editor.commands.get( 'strike' );
+		const keystroke = 'CTRL+E';
```

Comment:
Please advice of a good keystroke for that feature. I tried CTRL+ALT+S but it doesn't seems to work.

Reply:
cc @Comandeer 

I don't know why ctrl+alt+s didn't work so maybe there's some problem with https://github.com/ckeditor/ckeditor5-utils/blob/master/src/keyboard.js#L81 ;|

Reply:
Editor Recommendations does not define shortcut for strikethorugh, however it's probably good to stick to what Word does as it's a _de facto_ standard: <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>X</kbd> in Office 365 on macOS.

Reply:
@Reinmar @Comandeer what font do y'all use for icons?

Reply:
Can I use Cmd+Shift+X in the keystroke var here? Will the editor understand Cmd or should I use Meta?

Reply:
Right now typing Cmd key remove the selected content.

Reply:
@Natim Should be `Meta` I think

Reply:
Not according to https://github.com/ckeditor/ckeditor5-utils/blob/master/src/keyboard.js#L25